modules/urlsigning-service-impl/src/main/java/org/opencastproject/security/urlsigning/WowzaResourceStrategyImpl.java

```diff
@@ -39,15 +39,29 @@
   private static final String FIND_STREAM_FORMAT = ".{3}[:].*$";
   /** The URI scheme that an RTMP address uses. */
   private static final String RTMP_SCHEME = "rtmp";
+  /** The URI scheme that an http address uses. */
+  private static final String ADAPTIVESTREAMING_HTTP_SCHEME = "http";
```

Comment:
We removed the "adaptive" in most cases a while ago (see #1179), so just calling it streaming here would be fine.